build.gradle.kts

```diff
@@ -157,6 +157,7 @@ dependencies {
 
     // https://mvnrepository.com/artifact/org.mockito/mockito-all
     testImplementation("org.mockito:mockito-all:1.10.19")
+    testImplementation("org.mockito.kotlin:mockito-kotlin:5.1.0")
```

Comment:
Why do we need `mockito-kotlin`? For unit tests of our implementation?

Reply:
When we start using the implemented Kotlin support, we'll need `Mockito-Kotlin`. Therefore, I added it in advance.